src/moin/scripts/migration/moin19/macros/PageList.py

```diff
@@ -12,15 +14,16 @@
 def convert_page_list_macro_to_item_list(node):
     """ Convert the given PageList macro node to an ItemList macro in-place
 
-    Depending on the argument of the moin1.0 PageList macro, the new
-    ItemList macro will have a simple "item" argument or a "regex" argument.
+    The moin1.0 PageList macro used to pass all arguments to the FullSearch
+    macro, so they were essentially all treated as regular expression search queries.
+    After conversion to the ItemList macro, the argument will be a simple "regex" argument.
 
     Example conversions:
 
     | PageList macro (moin1.9)       | ItemList macro (moin2)           |
     |--------------------------------|----------------------------------|
     | <<PageList()>>                 | <<ItemList()>>                   |
-    | <<PageList(SomeSubPage)>>      | <<ItemList(item="SomeSubPage")>> |
+    | <<PageList(SomeSubPage)>>      | <<ItemList(regex="SomeSubPage")>> |
     | <<PageList(regex:Rnd[^abc]+)>> | <<ItemList(regex="Rnd[^abc]+")>> |
```

Comment:
ItemList macro displays matched subpages by default, PageList displays all matched pages. To make ItemList work like PageList, I think ItemList needs `item=""` parameter.

| <<PageList(Help)>> | <<ItemList(item="",regex="Help")>> |

Reply:
Good point, thanks for pointing it out. I changed the migration accordingly and hope the converted macros now behave the same as in version 1.9.